### DISCUSSION TITLE
How do I define Taxes? It would be great to have a list of taxes which I could attach to certain products or assortments

### DISCUSSION THREAD

**Discussion by nabbl** [unchainedshop/unchained#354] [Q&A] [answered]
Switzerland has different taxes for different kind of products. Right now if I set country and currency to swiss it calculates some taxes. But what if I want to sell food? Then I have a different tax.

Any way to set these?

**Comment by pozylon** [comment, marked as answer]
Yes, for Swiss tax you can use tags on products to achieve this:

https://github.com/unchainedshop/unchained/blob/master/packages/core-pricing/plugins/product-swiss-tax.js

if you set a tag swiss-tax-category:reduced it will be 0.025 for that product and if you set swiss-tax-category:special it will be 0.037

**Reply by pozylon** [reply]
Further if you have delivery fees and need to set it to one of the special categories, you can set a configuration key/value pair for that delivery provider

for ex. key swiss-tax-category
value special

-> delivery fees will be taxed 0.037

**Reply by nabbl** [reply]
very well .thank you very much!

**Comment by pozylon** [comment]
Taxes are highly specific to the delivery country, if you need any more features on product-swiss-tax or you need taxation rules for other countries please just copy the plugin file, load it locally, change it and send the changes as PR

**Reply by nabbl** [reply]
thank you. swiss taxes are fine for now :)
